### ISSUE TITLE
Mistake in code

### ISSUE THREAD

**Issue by Enyby** [SergioCrisostomo/Mootools-chart-flot#21]
https://github.com/SergioCrisostomo/Mootools-chart-flot/blob/master/Source/moo.flot.js#L712

In this line used `getStyle` but need `setStyle`.

**Comment by SergioCrisostomo** [comment]
@Enyby good catch! 
If you have possibility please send a fix for it, since you probably already fixed it in your code.

**Comment by Enyby** [comment]
Fix? Just change one letter. `getStyle` replace with `setStyle`.

**Comment by Enyby** [comment]
```
            if (placeholder.getStyle("position") == 'static')
                placeholder.getStyle("position", "relative"); // for positioning labels and overlay
```
must be
```
            if (placeholder.getStyle("position") == 'static')
                placeholder.setStyle("position", "relative"); // for positioning labels and overlay
```

**Comment by SergioCrisostomo** [comment]
@Enyby  I know that. Just wondered if you had a pull request ready, but no problem I fixed it in 1eb546908b9dbfbb04e7e5b4c0464853077a6818.

Thanks for reporting!
